src/main/java/seedu/address/logic/Messages.java

```diff
@@ -47,4 +48,19 @@ public static String format(Student student) {
         student.getTags().forEach(builder::append);
         return builder.toString();
     }
+
+    /**
+     * Formats the {@code moduleTiming} for display to the user.
+     */
+    public static String format(ModuleTiming moduleTiming) {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(moduleTiming.getModuleCode())
+                .append("; Day: ")
+                .append(moduleTiming.getDay())
+                .append("; Timing: ")
+                .append(moduleTiming.getStartTime())
+                .append("-")
+                .append(moduleTiming.getEndTime());
+        return builder.toString();
```

Comment:
It might be useful to add tests for this since its easy to miss regressions. Should be relatively quick to do, lmk if you're low on bandwidth and I can take it up.

Reply:
Sure, I'll review your tests 😃!